### DISCUSSION TITLE
Physics Hands Example Scene

### DISCUSSION THREAD

**Discussion by LeeVermeulen** [leapmotion/UnityModules#1305] [Q&A] [answered]
Physics hands is great! Looking forward to using it for a project

There is currently a pretty bare example scene - is the scene shown in the latest videos aviaable anywhere? Videos like https://www.youtube.com/watch?v=Ip1khhJdQGk&ab_channel=Ultraleap

**Comment by MattGrayUL** [comment, marked as answer]
Hi @LeeVermeulen

Glad you're enjoying Physics Hands!

The example scenes we provide with the Plugin are intended to be as simple as we can to allow you to learn what is suitable for each feature, this is particularly true in the Preview package (where Physics Hands lives)

The scene in the video you link builds upon the existing example scene by adding new meshes, but has no extra functionality than what the Example scene is capable of (in regards to physics hands). So feel free to take the example scene and then add whatever meshes and colliders you wish and see how Physics Hands works for you 🙂 

Finally, we are unable to make the scene from that video publicly available as it contains some assets that we do not have the license to share the source of, but @rorygames may be able to shed some more light on that 🙂

**Reply by LeeVermeulen** [reply]
No worries! Completely understandable - just wanted to check to save myself some time with putting a quick demo together.

**Comment by rorygames** [comment]
Hey @LeeVermeulen 

The scene I've been working on in the videos I post is currently closed source due to the numerous models and a specific plugin I make use of for [improved mesh colliders](https://assetstore.unity.com/packages/tools/level-design/non-convex-mesh-collider-automatic-generator-117273). 😄 

I'm looking at making a separate, open source version of my playground in the future, however it requires other work beyond just Physics Hands to make it possible 🥲.
